apps/docs/content/guides/auth/auth-email.mdx

```diff
@@ -157,6 +182,15 @@ Future<void> signOut() async {
 }
 ```
 
+</TabPanel>
+<TabPanel id="swift" label="Swift">
+
+When your user signs out, call [signOut()](/docs/reference/swift/auth-signout) to remove them from the browser session and any objects from localStorage:
```

Comment:
I know nothing about Swift so this may be a stupid question. Just double-checking that you do in fact refer to the browser and localStorage given it's a native app? (or is this just an artifact of copying the js one?)